docs/pbc/all/search/202410.0/base-shop/third-party-integrations/algolia/configure-algolia.md

```diff
@@ -159,11 +165,23 @@ Default attributes for faceting are as follows:
 - `label`
 - `prices`
 - `rating`
+- `merchant_name` (for Marketplace shops)
 
 The `prices` attribute is an object with nested fields. Algolia creates facets for each nested field and creates facets for all the currencies and pricing modes available in product entities.
 
 #### Facet configuration
 
+##### Facet display
+
+All configured index Facets will be displayed in the list of filter on Yves or via Glue API `/catalog-search`.
+Also, Spryker's Algolia app supports `renderingContent` feature, which can be found in an index "Configuration > Facet display".
```

Comment:
Is `Configuration > Facet display` in Algolia dashboard on back office?

Reply:
Algolia Dashboard in Search index Configuration